### PR TITLE
Adds wild yeast, fixing yeast times

### PR DESCRIPTION
**Pull request by Lamandus**
#### Summary
SUMMARY: Content "Adds wild yeast, fixing yeast times"

#### Purpose of change
This adds yeast recipe with jar, and wild yeast recipe, which can be found in books as well. Yeast times has been fixed (also wrong in DDA)

#### Testing
Works now as intended

### CODE REVIEW

File: data/json/recipes/recipe_medsandchemicals.json [hunk 1]
```diff
@@ -712,7 +712,45 @@
...
+  {
+    "type": "recipe",
+    "result": "wild_yeast",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "skills_required": [ "survival", 3 ],
+    "time": "20 m",
+    "batch_time_factors": [ 83, 5 ],
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_medsandchemicals.json:744]: Why 83?

File: data/json/recipes/recipe_medsandchemicals.json [hunk 2]
```diff
@@ -712,7 +712,45 @@
...
+    "book_learn": [ [ "brewing_cookbook", 2 ], [ "textbook_survival", 2 ] ],
+    "autolearn": true,
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_medsandchemicals.json:746]: Would be better to have the recipe be very easy, but autolearn hard.

File: data/json/recipes/recipe_medsandchemicals.json (hunk 1 above)
Comment (Lamandus) [reply data/json/recipes/recipe_medsandchemicals.json:744]: old take over, I change to 50, 3. a bit of heating is necessary.

File: data/json/recipes/recipe_medsandchemicals.json (hunk 2 above)
Comment (Lamandus) [reply data/json/recipes/recipe_medsandchemicals.json:746]: good idea

**Pull request merged**